packages/core/watcher/src/child.js

```diff
@@ -18,6 +18,8 @@ function init(options) {
   options = optionsTransfer.decode(options);
   watcher = new FSWatcher(options);
   watcher.on('all', sendEvent);
+  // only used for testing
+  watcher.on('ready', () => sendEvent('_chokidarReady'));
```

Comment:
Might be a good idea to put this comment on the same line if the line below is needed outside of testing.